src/NzbDrone.Core/Notifications/Pushsafer/InvalidResponseException.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace NzbDrone.Core.Notifications.Pushsafer
+{
+    public class InvalidResponseException : Exception
```

Comment:
Not seeing this used anywhere.

Reply:
It can be safely removed as it's not used. PushSaferService actually reports to the logger every exception. That should be sufficient.